02-spatial-data.Rmd

```diff
@@ -802,7 +796,7 @@ st_crs(sf_points)
 ```
 
 ## Units
-
+<!-- If this is the final thing about sf, we should move it up before starting to explain raster stuff -->
```

Comment:
I was thinking we could talk about units in a way that is relevant to raster and vector data. With raster data the most important thing is the resolution and (I don't think) the **raster** package has native support for units. I think that's worth mentioning at the end of the chapter. So I suggest:

1. An introductory paragraph on the importance of units for geographic data.
2. A few paragraphs explaining units in **sf**, e.g. beginning with a sentence like `An innovative feature of **sf** compared with its predecessor **sp** is that it has *native support* for units.` followed by some worked examples.
3. A paragraph or 2 on units in raster data with a worked example.